lib/puppet/pops/types/puppet_object.rb

```diff
@@ -11,6 +11,16 @@ module PuppetObject
   def _ptype
```

Comment:
Changing that will break the netconf stuff. They use _ptype in several places. Using "_p" as a prefix works for me. My preference would be to keep it as is (including the other methods).